<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditUsername">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDiscipline">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Bioquímica</string>
    </property>
    <item>
     <property name="text">
      <string>Bioquímica</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Físico-química</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Química Geral</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1º Bimestre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Disciplina</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_n2Second">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>267</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>277</y>
      <width>53</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Nota 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>227</y>
      <width>53</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Nota 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_n1Second">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>219</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_n1First">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_n2First">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>227</y>
      <width>53</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Nota 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>178</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2º Bimestre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrint">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Imprimir Notas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddStudent">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadastrar  Aluno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>277</y>
      <width>53</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Nota 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCheckTest">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserir  Substutiva</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
